index.html

```diff
@@ -2247,6 +2091,175 @@
+violate the principles around use of information acquired in a different
+[=context=]. This is particularly true for [=vulnerable=] people, as
+recognising them in different [=contexts=] may force traits into the open
+that reveal their vulnerability. For example, if you meet your therapist at a
+party, you expect them to have different discussion topics with you than they
+usually would, and possibly even to pretend they don't know you.
+
+<dfn>Cross-site recognition</dfn> is when a [=site=] determines with high
+probability that a visit to the site comes from the same person as another
+visit to a *different* site. In the usual case that the sites are different [=contexts=],
+[=cross-site recognition=] is a privacy harm in the same cases as [=cross-context recognition=].
+
+<dfn>Same-site recognition</dfn> is when a single [=site=] discovers and uses the
+fact that two or more visits probably came from the same [=person=].
+
+A privacy harm occurs if a [=person=] reasonably expects that they'll be using
+a different [=identity=] for different visits to a single site, but the site
+[=recognizes=] them anyway. This harm can be accomplished through a
+variety of means detailed in [[[#recognition-methods]]].
+
+Note that these categories overlap: [=cross-site recognition=] is usually
+[=cross-context recognition=] (and always [=recognizes=] across [=partitions=]); and
+[=same-site recognition=] is sometimes [=cross-context recognition=] (and may or may not
+involve multiple [=partitions=]).
+
+### User agent awareness of recognition {#user-agent-recognition}
+
+A <dfn>partition</dfn> is the [=user agent=]'s attempt to match how its user
+would understand a [=context=]. [=User agents=] don't have a perfect
+understanding of how their users experience the sites they visit, so they
+often need to approximate the boundaries between [=contexts=] when building
+[=partitions=].
+
+In the absence of better information, a [=partition=] can be defined as:
+
+* a set of [=environments=] (roughly: same-site and cross-site [^iframe^]s,
+workers, and top-level pages)
+* whose [=environment/top-level origins=] are in the [=same site=] (note: see
+[[PSL-Problems]])
+* being visited within the same user agent installation (and browser profile,
+container, or container tab for user agents that support those features)
+* between points in time that the person or user agent clears that [=site=]'s
+cookies and other storage (which is sometimes automatic at the end of each
+session).
+
+It can be difficult for a [=user agent=] to detect when a single site contains
+multiple [=contexts=]. When a [=user agent=] can detect this, it should adjust
+its [=partitions=] accordingly, for instance by partitioning identities
+per subdomain or site path. User agents should work to improve their ability to
+distinguish contexts within a site.
+
+[=User agents=] should prevent people from being [=cross-context
+recognition|recognized=] across [=partitions=] unless they intend to be
```

Comment:
I think the `data-lt` above lets us:

```suggestion
[=User agents=] should prevent people from being
[=recognized=] across [=partitions=] unless they intend to be
```